kubernetes/filters.go

```diff
@@ -71,6 +71,22 @@ func FilterDestinationRulesByHostname(allDr []networking_v1alpha3.DestinationRul
 	return destinationRules
 }
 
+func FilterDestinationRulesByNamespaces(namespaces []string, allDr []networking_v1alpha3.DestinationRule) []networking_v1alpha3.DestinationRule {
+	destinationRules := []networking_v1alpha3.DestinationRule{}
+	for _, dr := range allDr {
+		found := false
+		for _, ns := range namespaces {
+			if dr.Namespace == ns {
+				found = true
```

Comment:
I think better to add "break" here when namespace matches.

Reply:
And just a note :) Python allows to have this whole function in one line :)

Reply:
+1 yeah, I miss some functional helpers in Golang for these loops. Thanks !